Design rules/Policy Design Rules.md

```diff
@@ -0,0 +1,42 @@
+3. Tevens houden we ons aan [de Design Rules van VNG Realisatie](https://github.com/VNG-Realisatie/API-Kennisbank/blob/master/Design%20rules/readme.md);
+4. Tenslotte kijken we ook in hoeverre de Extensions op de Design Rules van VNG Realisatie van toepassing zijn voor het project. Ook hier bschrijven we in een Design Decision van het project weer aan welke Extensions we ons houden;
+
+## Het toevoegen, wijzigen of verwijderen van VNG Realisatie Design Rules
+
+Zoals al gesteld in [de Design Rules van VNG Realisatie](https://github.com/VNG-Realisatie/API-Kennisbank/blob/master/Design%20rules/readme.md) kunnen Design Rules onderhevig zijn aan voortschrijdend inzicht en technologische ontwikkelingen. 
+Het kan dus noodzakelijk blijken een Design Rule te wijzigen of verwijderen. Ook kan er behoefte aan een nieuwe Design Rule ontstaan. Over het algemeen zal deze behoefte in een project naar voren komen. 
+In dat geval handelen we als volgt:
+
+1. Er wordt in de onderliggende GitHub repository een issue ingediend met het label 'Design Rule'. 
+2. Indien het om een voorstel voor een nieuwe Design Rule gaat wordt deze zo duidelijk mogelijk beschreven, zo mogelijk voorzien van voorbeelden en ratio;
+3. Indien het om een voorstel tot wijziging gaat wordt beschreven welke Design Rule moet worden gewijzigd, wat die wijziging inhoudt en wat daarvoor de ratio is;
+4. Betreft het een verwijderingsvoorstel dan wordt beschreven welke Design Rule het betreft en wat de reden is van het verzoek;
+5. Het issue wordt assigned aan alle medewerkers van het Kenniscentrum Standaarden;
+6. Het issue wordt geagendeerd als onderwerp van het teamoverleg Kenniscentrum Standaarden waar wordt besloten of het voorstel wordt gehonoreerd. 
+In de periode tussen het opvoeren van het issue en het besluit wordt in de projecten niet vooruitgelopen op dat besluit. 
+Aanpassingen n.a.v. het honoreren van het voorstel worden dus pas aangebracht nadat het voorstel gehonoreerd is.
+
+## Voordracht VNG Realisatie Design Rules voor Landelijke API Strategie
+
+Design Rules van VNG Realisatie kunnen door het Kenniscentrum Standaarden worden aangeboden aan de werkgroep 'API design rules' van het Kennisplatform API's.
+Eenieder kan daartoe een verzoek indienen. De te volgen procedure is daarbij als volgt:
+
+1. Er wordt in de onderliggende GitHub repository een issue ingediend met het label 'Design Rule'.
+2. Er wordt beschreven om welke Design Rule het gaat en waarom men van mening is dat de betreffende Design Rule opgetild moet worden naar landelijk niveau.
+3. Het issue wordt assigned aan alle medewerkers van het Kenniscentrum Standaarden;
+4. Het issue wordt geagendeerd als onderwerp van het teamoverleg Kenniscentrum Standaarden waar wordt besloten of het voorstel wordt gehonoreerd. 
```

Comment:
Ook hier tussen 3 en 4 eerst proberen de discussie langs digitale weg te beslechten. Pas agenderen in een overleg als we er niet uitkomen.